pkg/services/ngalert/api/tooling/post.json

```diff
@@ -838,6 +840,16 @@
      },
      "type": "array",
      "x-go-name": "Alertmanagers"
+    },
+    "alertmanagersChoice": {
+     "enum": [
+      "all",
+      "internal",
+      "external"
+     ],
+     "type": "string",
+     "x-go-enum-desc": "all AllAlertmanagers\ninternal InternalAlertmanager\nexternal ExternalAlertmanagers",
+     "x-go-name": "AlertmanagersChoice"
```

Comment:
Please make sure you update the spec file, and we mention that the default is "all"